tests/acceptance/testall

```diff
@@ -389,6 +389,18 @@ export CFENGINE_TEST_OVERRIDE_WORKDIR TEMP CFENGINE_TEST_OVERRIDE_EXTENSION_LIBR
 
     TEST_END_TIME=$(unix_seconds)
 
+    # Try to collect test metadata if any
+    if egrep -e "R: test description: " $OUTFILE > /dev/null; then
+      TEST_DESCRIPTION="$(egrep -e "R: test description" $OUTFILE | sed -e "s,.*test description: \(\w*\),\1,")"
+    fi
+    if egrep -e "R: test story_id: " $OUTFILE > /dev/null; then
+      TEST_STORY="$(egrep -e "R: test story_id" $OUTFILE | sed -e "s,.*test story_id: \([0-9][0-9]*\),\1,")"
+    fi
+    if egrep -e "R: test covers: " $OUTFILE > /dev/null; then
+      TEST_COVERS="$(egrep -e "R: test covers" $OUTFILE | sed -e "s,.*test covers:[ \t][ \t]*\(\w*\),\1,")"
```

Comment:
``\w`` is not in POSIX. I'm pretty sure the non-Linux sed tools won't understand it.

The same with ``egrep -e``. Just use ``egrep`` without the ``-e``.